_quarto.yml

```
@@ -18,9 +18,20 @@ website:
   sidebar: 
    contents:
     - index.md
+    - section: "Overview"
```

The overview is not showing up, shouldn't it be:

```suggestion
    - "overview/index.md"
```